dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityChangeLogServiceTest.java

```diff
@@ -218,6 +219,8 @@ void shouldReturnChangeLogsWhenTrackedEntityAttributeValueIsCreated() throws Not
     programOwnerService.createOrUpdateTrackedEntityProgramOwner(
         trackedEntity.getUid(), program.getUid(), orgUnitA.getUid());
 
+    createAndPersistProgramAttribute(program);
+
```

Comment:
this way of using helpers is confusing to me. you say `createAndPersistProgramAttribute` by passing the program. But then in the method you have the `trackedEntityAttribute` of the class scope. In my opinion, say you want to create program attributes, i would do a method like `createProgramAttributes` or `programAttributes` by passing the program and the attributes. At a first glance, i am not interested in what is going on inside the method, but i want to see a signature explaining what is going on